packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component.ts

```diff
@@ -201,7 +201,7 @@ export class NgxFlickingComponent implements OnInit, AfterViewInit, OnDestroy, O
     }
 
     this.internalCloneCount = newCC;
-    this.cdr.detectChanges();
+    this.cdr.markForCheck();
```

Comment:
What's the reason using `markForCheck` instead of `detectChanges`?

Reply:
@WoodNeck 

markForCheck(): Marks all ChangeDetectionStrategy ancestors as to be checked.
detectChanges(): Checks the change detector and its children.

https://medium.com/@pablo.platt/can-you-please-explain-when-to-use-markforcheck-and-when-detectchanges-52e281783c6

Reply:
I wanted you to explain the reason why you've chosen `markForCheck`, not definition.
Looks like this link explains good about it.
https://stackoverflow.com/questions/41364386/whats-the-difference-between-markforcheck-and-detectchanges/45396740#41364469